android-components/components/feature/media/src/test/java/mozilla/components/feature/media/fullscreen/MediaSessionFullscreenFeatureTest.kt

```diff
@@ -370,6 +371,8 @@ class MediaSessionFullscreenFeatureTest {
         store.waitUntilIdle()
         verify(activity.window).clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
 
+        clearInvocations(activity.window)
```

Comment:
Nice catch!
If we're resetting the `activity.window` state then the `atLeastOnce()` from below can probably be avoided also.

Reply:
Thanks, I remove `atLeastOne`